adsft/extraction.py

```diff
@@ -393,14 +391,11 @@ def open_xml(self):
             with open(self.file_input, 'rb') as fp:
                 raw_xml = fp.read()
 
-            # use soupparser to properly encode file contents
-            #  it could be utf-8, iso-8859, etc.
-            parsed_content = soupparser.fromstring(raw_xml)
-            # convert to string for ease of clean-up, convert html and LaTeX entities
-            raw_xml = tostring(parsed_content)
-            raw_xml = re.sub('(<!-- body|endbody -->)', '', raw_xml)
+            encoding = UnicodeDammit(raw_xml).originalEncoding

```

Comment:
I guess it would be good to keep/write some comments so that we will remember in the future what are we doing with these two lines.

Reply:
Just added some.